Source/Scene/Scene.js

```diff
@@ -2727,6 +2799,9 @@ define([
         var uniformState = context.uniformState;
 
         var drawingBufferPosition = SceneTransforms.transformWindowToDrawingBuffer(this, windowPosition, scratchPosition);
+        if (this.pickTranslucentDepth) {
+            renderForPickDepth(this, drawingBufferPosition);
```

Comment:
>Are you sure pickTranslucentDepth need to be a member of Scene?

Yes, unless you have a better suggestion. We call `pickPosition` in `ScreenSpaceCameraController` for panning, zooming, terrain collision, etc. If we don't have the option on Scene, would we do it by default?